src/pudl/etl/__init__.py

```diff
@@ -37,6 +37,12 @@
     *load_assets_from_modules([glue_assets], group_name="glue"),
     *load_assets_from_modules([pudl.extract.eia860], group_name="eia860_raw_assets"),
     *load_assets_from_modules([pudl.extract.eia923], group_name="eia923_raw_assets"),
```

Comment:
Why do some of the `group_name`s end with `_assets` while others do not (e.g. `eia860_raw_assets` vs. `glue`)?

Why do some of the `group_name`s reflect the name of the data source (like `epacems`) while others do not (e.g. `eia_api` rather than `eia_bulk_elec` -- there are many different EIA APIs, the bulk electricity data is just from one of them)?

Why is there a direct correspondence between the module name and the group name for some asset groups, and not for others?

Reply:
Ah sorry this comment slipped through the cracks. Yeah, I've been fast and loose with the asset group naming convention. Sounds like you might be standardizing the pattern in a separate PR?